robot_sdk/wrappers/ec_wrapper.py

```
@@ -157,38 +169,48 @@ def move_in_arc(self, point_1: List[float], point_2: List[float]) -> Optional[bo
+            # Convert m to mm
+            new_pose[0] += x*1000
+            new_pose[1] += y*1000
+            new_pose[2] += z*1000
+            if tool_vel is None:
+                tool_vel = 0.20
+            if tool_acc is None:
+                tool_acc = 0.01
+
+            final_target = self.client.user_to_base_cord_conversion(user_num=user_num, target=new_pose)
+            suc, result = self.client.movel(points=final_target, tool_vel=tool_vel, tool_acc=tool_acc)            
+            return suc, result
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)